_includes/research-platform/projects-collapsable.html

```diff
@@ -0,0 +1,35 @@
+<section id="projects" class="container content-section">
+    <div class="row">
+        <div class="col-lg-12">
+            <p>
+                We aim to connect students and research leaders in ML/AI 
```

Comment:
Maybe research labs?

Reply:
That phrase was in the original About section I guess it's like this because "research labs" is already used before and after this sentence.